requirements-py3.txt

```diff
@@ -0,0 +1 @@
+testtools>=0.9.22
```

Comment:
This file isn't needed. Mock is installable on python3 with no harmful side effects - its simpler to just have the one requirements file.